server/container_create.go

```diff
@@ -766,9 +766,6 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	specgen.SetRootPath(mountPoint)
 
 	saveOptions := generate.ExportOptions{}
-	if err = specgen.SaveToFile(filepath.Join(containerInfo.Dir, "config.json"), saveOptions); err != nil {
```

Comment:
don't we need this in `Dir` when rebooting a machine? `RunDir` will go away? (can't fully check what I'm saying /cc @nalind ). Point I think is if we don't save config in /var/lib, we can't restore containers